service/server.py

```diff
@@ -278,11 +278,31 @@ def format_proprietors(proprietors_data):
         name = proprietor.get('name') or ''
         addresses = proprietor.get('addresses') or []
         formatted_proprietor = {}
-        # TODO: proprietor names have potentially a lot more fields to display
-        if 'forename' in name and 'surname' in name:
-            formatted_proprietor["name"] = name['forename'] + ' ' + name['surname']
+        if 'name_information' in name:
+            formatted_proprietor["name_information"] = ', ' + name['name_information']
+        if 'name_supplimentary' in name:
+            formatted_proprietor["name_supplimentary"] = ', ' + name['name_supplimentary']
+        if 'charity_name' in name:
+            charity_name = ' of '
+            if name['charity_name'].endswith(")"):
+                charity_name += '('
+            charity_name += name['charity_name']
+            formatted_proprietor["charity_name"] = charity_name
+        if 'trading_name' in name:
+            formatted_proprietor["trading_name"] = ' trading as ' + name['trading_name']
+        if 'forename' in name or 'surname' in name:
+            formatted_proprietor["name"] = format_pi_name(name)
         if 'non_private_individual_name' in name:
             formatted_proprietor["name"] = name['non_private_individual_name']
+            if 'company_reg_num' in name:
+                formatted_proprietor["co_reg_no"] = 'Company registration number ' +\
+                                                    name['company_reg_num']
+            if 'company_location'in name and 'country_incorporation' in name:
+                if name['company_location'] != '':
```

Comment:
There should always be both. I got a bit carried away as I think there'll be more stories around company_location as scottish and irish ones don't have company_incorporation but are still flagged as not English/Welsh. 
I'll remove the extra company_incorporation as not needed for the story.